### PR TITLE
Add possibility to generate iban for specific country

### PR DESCRIPTION
**Pull request by nning**

### CODE REVIEW

**Comment by Moosh-be** [comment]
@nning  the code fails on very old versions (node 0.10 0.11 and 0.12 )
 https://github.com/Marak/faker.js/blob/master/.travis.yml#L17

**Comment by Moosh-be** [comment]
@nning  the code fails on very old versions (node 0.10 0.11 and 0.12 )
 https://github.com/Marak/faker.js/blob/master/.travis.yml#L17 
I can't test it on my Synology 
Perhpas I can  learn how to use docker :)

File: lib/finance.js [hunk 1]
```diff
@@ -233,8 +233,19 @@ var Finance = function (faker) {
...
    * @method  faker.finance.iban
    */
-  self.iban = function (formatted) {
-      var ibanFormat = faker.random.arrayElement(ibanLib.formats);
+  self.iban = function (formatted, country) {
+      var ibanFormat;
+      if (typeof country === 'undefined') {
+        ibanFormat = faker.random.arrayElement(ibanLib.formats);
+      } else {
+        ibanFormat = ibanLib.formats.filter((format) => {
```
Comment (Marak) [inline lib/finance.js:241]: @nning -

Sorry to nitpick, could you remove the fat arrow here? 

We don't allow fat arrows in `faker.js`.

Re: #841

**Comment by shakedlokits** [comment]
This nitpick has stuck this required feature for the past year

**Comment by nning** [comment]
Already fixed in `master`, right?

**Pull request closed without merging**